sockeye/inference.py

```diff
@@ -452,31 +460,40 @@ class ModelState:
     A ModelState encapsulates information about the decoder states of an InferenceModel.
     """
 
-    def __init__(self, states: List[mx.nd.NDArray]) -> None:
+    def __init__(self, states: List[mx.nd.NDArray]=None) -> None:
```

Comment:
Why?

Reply:
Why the default argument? The constructor is used (with the default argument) to pre-fill an array later.

Reply:
I see. Maybe this won't be needed when encoding is batched as well.